board.py

```diff
@@ -1,49 +1,36 @@
 class Board:
-    def __init__(self):
-        self.board = {
-            "1": "1",
-            "2": "2",
-            "3": "3",
-            "4": "4",
-            "5": "5",
-            "6": "6",
-            "7": "7",
-            "8": "8",
-            "9": "9",
-        }
-
-    def draw_board(self):
-        # draws the current board state
-        divider = "+---+---+---+"
-        print(divider)
-        print(
-            f"| {self.access_board('1')} | {self.access_board('2')} | {self.access_board('3')} |"
-        )
-        print(divider)
-        print(
-            f"| {self.access_board('4')} | {self.access_board('5')} | {self.access_board('6')} |"
-        )
-        print(divider)
-        print(
-            f"| {self.access_board('7')} | {self.access_board('8')} | {self.access_board('9')} |"
-        )
-        print(divider)
+    def __init__(self, size):
+        self.board = {}
+        self.size = size
+        self.highest_value = size * size
+        self.generate_board()
+        self.arrangements = self.generate_arrangements()
 
+    # controls board access - if it creates, reads, or writes to the game board, it goes through this
     def access_board(self, value, new_value=None):
```

Comment:
I find `access_board` a little confusing for our domain. What am I accessing? In the tests it suggests that we 'access' the board to, essentially, make/play a move.

If there's methods for making a move and another use case, is that a hint that we're asking our method to do too much and giving it multiple responsibilities?

In the same vein, I find `value` a bit of a challenge to grok. Value is quite broad - in my opinion it's synonymous with 'argument' or 'parameter'. What could it be here's that's _not_ a value?